migrations/main.sql

```diff
@@ -633,3 +633,27 @@ CREATE VIEW user_request_booking_hostel_view AS
     ORDER BY
         u.university_id,
         s.user_id;
+
+
+DROP VIEW IF EXISTS return_user_document_view;
+CREATE VIEW return_user_document_view AS
```

Comment:
u'll get an error too soon bcs the columns order of your SQL view, sqlalchemy view obj and out schema aren't the same. Please make it all identical
Select id's column firstly...

Reply:
Hmmm, before creating pr I tester it, so it worked
Can u check it pls, or maybe I can't understand where is mistake?